vector/src/main/java/im/vector/app/features/login/qr/QrCodeLoginViewModel.kt

```diff
@@ -58,20 +76,60 @@ class QrCodeLoginViewModel @AssistedInject constructor(
     }
 
     private fun handleOnQrCodeScanned(action: QrCodeLoginAction.OnQrCodeScanned) {
-        if (isValidQrCode(action.qrCode)) {
-            setState {
-                copy(
-                        connectionStatus = QrCodeLoginConnectionStatus.ConnectingToDevice
-                )
+        Timber.tag(TAG).d("Scanned code: ${action.qrCode}")
```

Comment:
I would remove this log, I think it can contain sensitive data. Logs can ends up in rageshake, so we have to be careful about that. See https://github.com/vector-im/element-android/blob/main/docs/_developer_onboarding.md#logging for more details.

Reply:
For our use case the QR doesn't actually contain data that cannot be disclosed. However, I take the general point and am removing it.

Reply:
For our use case the QR doesn't actually contain data that cannot be disclosed. However, I take the general point and am removing it.